### DISCUSSION TITLE
It would be really cool if the server would save what shows a person has watched so it's synced across devices

### DISCUSSION THREAD

**Discussion by jordyvandomselaar** [jellyfin/jellyfin#5306] [General] [answered]
Right now Jellyfin keeps track of what I've watched on every device, it would be really cool if watching a show on my phone also updates the shows I've watched on my TV or PC.

**Comment by crobibero** [comment, marked as answer]
I'm not sure what your configuration is like, but this already happens

**Reply by jordyvandomselaar** [reply]
Oh 😂️ , I'm using the defaults and I watched a show on my PC but when I open the show on my phone it still says next up episode 1 of season 1

**Reply by Heciruam** [reply]
There are two sections. 
"Continue Watching" and "Next up". They are different so you might want to add both to your "home screen"

**Reply by jordyvandomselaar** [reply]
Makes sense, I'll give that a shot. Thanks!

**Reply by jordyvandomselaar** [reply]
So, it works on my TV but not on my PC when I launch Jellyfin to watch.
